src/client/nav/request.js

```diff
@@ -126,20 +126,20 @@ spf.nav.request.send = function(url, opt_options) {
       spf.config.get('experimental-request-headers'));
     if (configHeaders) {
       for (var key in configHeaders) {
-        headers[key] = configHeaders[key];
+        headers[key] = configHeaders[key] || '';  // Treat null, etc as empty.
```

Comment:
So, values _should_ be strings but I agree, that's probably an unexpected result.